docs/zh/development/apollo-development-guide.md

```diff
@@ -62,6 +62,21 @@ Apollo本地开发需要以下组件：
 
 ![ConfigAdminApplication-Eureka](https://raw.githubusercontent.com/ctripcorp/apollo/master/doc/images/local-development/ConfigAdminApplication-Eureka.png)
 
+> 注：除了在Eureka确认服务状态外，还可以通过健康检查接口确认服务健康状况：
+>
+> AdminService： [http://localhost:8090/health](http://localhost:8090/health)
+> ConfigService： [http://localhost:8080/health](http://localhost:8080/health)
```

Comment:
How about using `apollo-configservice` and `apollo-adminservice` in order to be consistent with other places in the document?

```suggestion
> apollo-adminservice： [http://localhost:8090/health](http://localhost:8090/health)
> apollo-configservice： [http://localhost:8080/health](http://localhost:8080/health)
```

Reply:
thanks for your suggest :)